aesara/_version.py

```diff
@@ -16,6 +16,7 @@
 import sys
 from typing import Dict
 
+
```

Comment:
```suggestion
```

@aseyboldt, `pre-commit` is set to ignore this file, so it's better to use `pre-commit run` instead of running `black` by hand.